schema/index.graphql

```diff
@@ -218,14 +331,24 @@ type TaxValue {
   taxClass: TaxClass!
 }
 
+"""
+An expense to an order/cart
+"""
+type Expense {
+  id: ID!
+  label: String
+  value: Int!
+}
+
 """
 Total amounts of a Cart or a CartItem
 """
 type Totals {
   net: Int!
   gross: Int!
   taxes: [TaxValue]!
-  discounts: [Discount!]!
+  discounts: [Discount!]
+  expenses: [Expense!]
```

Comment:
Expenses could be freight surcharges or an expense caused by choosing a different shippment or payment method.